agoradatatools/etl/transform.py

```diff
@@ -103,6 +103,11 @@ def calculate_distribution(df: pd.DataFrame, col: str, is_scored, upper_bound):
 def transform_overall_scores(df: pd.DataFrame) -> pd.DataFrame:
     interesting_columns = ['ensg', 'hgnc_gene_id', 'overall', 'geneticsscore', 'omicsscore', 'literaturescore']
 
+    # # fly_neuropath subtraction is required for v9 source; remove when we go to v13 source
+    df['overall'] = df['overall'] - df['flyneuropathscore'].astype(dtype='float64', errors='raise')
+    df.drop(columns=['flyneuropathscore'], inplace=True)
```

Comment:
Not familiar with the source files, is there any case where one or both of 'overall' or 'flyneuropathscore' have NA values inside? If so, do we need to account for that, or is ending up with NA from the subtraction ok?

Reply:
Ditto for the overall score (logsdon) data